README.rst

```diff
@@ -0,0 +1,83 @@
+
+
+================
+Laplace Equation
+================
+
+This example solves the weak form of the standard Laplace equation,
+
+|laplace_equation1|
+
+using the Galerkin Finite Element method. |conductivity_tensor| and |phi| are the rank two conductivity tensor and a scalar field (e.g. temperature) respectively. A homogeneous and isotropic material with |equation1| (identity tensor) is considered in this example. As a result, |conductivity_tensor| in the above equation becomes redundant and the equation reduces to
+
+|laplace_equation2|.
+
+Only Dirichlet boundary conditions are prescribed.
+
+.. |laplace_equation1| image:: ./images/laplace_equation1.svg
+   :align: middle
+
+.. |conductivity_tensor| image:: ./images/conductivity_tensor.svg 
+   :align: middle
+
+.. |phi| image:: ./images/field_scalar.svg
+   :align: middle
+   
+.. |equation1| image:: ./images/equation1.svg
+   :align: middle
+
+.. |laplace_equation2| image:: ./images/laplace_equation2.svg
+   :align: middle
+   
+Building the example
+====================
+
+The fortran version of the example can be configured and built with CMake::
+
+  mkdir examples
+  cd examples
+  git clone https://github.com/OpenCMISS-Examples/laplace_equation
+  cd laplace_equation
+  cmake -DOpenCMISSLibs_DIR=/path/to/opencmisslib/install
```

Comment:
and change this to `cmake -DOpenCMISSLibs_DIR=/path/to/opencmisslib/install ../laplace_equation`